releasenotes/notes/oxidize-synth-clifford-greedy-0739e9688bc4eedd.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade_synthesis:
+  - |
+    The function :func:`.synth_clifford_greedy` that synthesizes :class:`.Clifford` operators
+    was ported to Rust.
```

Comment:
it's worth to elaborate here on the performance improvement

Reply:
Done in [441c9f3](https://github.com/Qiskit/qiskit/pull/12601/commits/441c9f3a1a312320fdb2697a36073a1e16ad532d).